sep-template.md

```diff
@@ -40,3 +40,14 @@ objections or concerns raised during discussion.
 ## Security Concerns
 All SEPs should carefully consider areas where security may be a concern, and document them
 accordingly. If a change does not have security implications, write "N/A".
+
+## Changelog
+All SEPs should contain a changelog that succinctly lists its versions and what change was introduced in each version. Information included should contain a succinct by complete description of what has changed. Succinct is important so that over time a large changelog is easily digested.
+
+Example:
+
+- `v0.1.0`: Initial draft. [#1000](https://github.com/stellar/stellar-protocol/pulls/1000)
+- `v0.2.0`: Redesigned the API. [#1004](https://github.com/stellar/stellar-protocol/pulls/1004)
+- `v1.0.0`: Updated status to Active. [#1234](https://github.com/stellar/stellar-protocol/pulls/1234)
+- `v1.1.0`: Add new paremeters to the X endpoint. [#3498](https://github.com/stellar/stellar-protocol/pulls/3498)
+- `v1.1.1`: Fix misleading statements about the P parameter. [#3499](https://github.com/stellar/stellar-protocol/pulls/3499)
```

Comment:
🤔  I'd prefer ordering changes from newest to oldest, WDYT? It's the kind of order found in [npm](https://www.npmjs.com/package/stellar-sdk) and other platforms.

Reply:
Good idea. Changed in 9546e9a613ec3503cc5893be4cf0d3b1de3a8d4e.